catalog/admin.py

```diff
@@ -1,3 +1,38 @@
 from django.contrib import admin
+from .models import Author, Genre, Book, BookInstance
 
 # Register your models here.
+#admin.site.register(Book)
+#admin.site.register(Author)
+admin.site.register(Genre)
+#admin.site.register(BookInstance)
+class AuthorAdmin(admin.ModelAdmin):
```

Comment:
line 9-> 11, ko có custom gì thì em dùng cú pháp ngắn gọn để register thôi nhé
```suggestion
admin.site.register(Author)
```